packages/frontend/.env.production

```diff
@@ -0,0 +1,5 @@
+REACT_APP_NETWORK="mainnet"
```

Comment:
Why is this `.env.production` committed? Env vars should be set on netlify dashboard